lib/fixture/index.ts

```diff
@@ -0,0 +1,38 @@
+import {Fixtures} from '@playwright/test'
+import {
+  Queries as ElementHandleQueries,
+  queriesFixture as elementHandleQueriesFixture,
+} from './element-handle'
+import {
+  Queries as LocatorQueries,
+  queriesFixture as locatorQueriesFixture,
+  registerSelectorsFixture,
+  installTestingLibraryFixture,
+} from './locator'
```

Comment:
Maybe a bit OCD of me but I'd separate out the `import type` statements here, too.

Reply:
Haha, oh I feel ya on the OCD — don't hold back!